rts/lra/lra-proxy/api/src/main/java/io/narayana/lra/client/internal/proxy/nonjaxrs/LRACDIExtension.java

```diff
@@ -109,10 +111,25 @@ private LRAParticipant getAsParticipant(ClassInfo classInfo) throws ClassNotFoun
         return participant.hasNonJaxRsMethods() ? participant : null;
     }
 
-    private boolean isNotLRAParticipant(ClassInfo classInfo) {
-        Map<DotName, List<AnnotationInstance>> annotations = classInfo.annotations();
-        return !annotations.containsKey(DotNames.LRA) ||
-            (!annotations.containsKey(DotNames.COMPENSATE) &&
-                !annotations.containsKey(DotNames.AFTER_LRA));
+    /**
+     * Returns whether the classinfo represents an LRA participant --
```

Comment:
```suggestion
     * Returns whether the classinfo represents a LRA participant
```

Reply:
didn't we agree that it's read as (e)LRA? so it should be an LRA not a LRA?